### PR TITLE
Handle native querySelector returning nothing

### PR DESCRIPTION
**Pull request by jcmoore**
```
- Fixes #476
```

### CODE REVIEW

File: src/querySelector.js [hunk 1]
```diff
@@ -128,7 +128,11 @@
         return findOne(this, selector);
       }
 
-      if (root = getTreeScope(wrappedItem).root) {
+      if (!wrappedItem) {
+          // When the original query returns nothing
```
Comment (arv) [inline src/querySelector.js]: wrong indentation

**Comment by arv** [comment]
LGTM. Thanks for fixing.

Would you mind fixing the style nit and squash?

**Comment by jcmoore** [comment]
Should be all set now BTW,  thanks!

**Comment by arv** [comment]
Thanks again.

**Pull request merged**